packages/gafl-webapp-service/src/processors/refdata-helper.js

```diff
@@ -8,8 +8,9 @@ const optionProc = c =>
 
 const local = {}
 
-const fetch = async () =>
-  optionProc(await salesApi.countries.getAll())
+const fetch = async () =>{
+  // Returns all countries from Dynamics
+  return optionProc(await salesApi.countries.getAll())
     .sort((a, b) => {
       const order = ['GB-ENG', 'GB-WLS', 'GB-SCT', 'GB-NIR', 'GB']
```

Comment:
GB should be removed from here now - and the test will need to be updated

Reply:
Removing it here only causes the order to change, so 'United Kingdom' is moved to the top of the list, rather than below the individual countries, so I'm not sure that it matters whether it is in or not as we remove the option for the dropdown list anyway, so I'll remove it :)

Reply:
After our Slack chat, I have left this in